### PR TITLE
PKF-21: Inherit latest Haiti distribution

### PR DESCRIPTION
**Pull request by nuhuhmutebi**
All changes combined for the `openmrs-config`

### CODE REVIEW

File: configuration/concepts/activate_forms.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:2000
-,,,All Observation Templates,All Observation Templates,,ConvSet,N/A,,,Haiti_Vital Signs Form; History and Examination; Second Vitals; Vaccination History; Lab Samples; Radiology Orders,,,,,,,,
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Fully specified name:fr,Short name:fr,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:2000
+,,,All Observation Templates,All Observation Templates,,All Observation Templates,All Observation Templates,,ConvSet,N/A,,,Echantillons de laboratoire; Ordonnances de radiologie,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/activate_forms.csv]: Fix the uuid for the concept similar to how it is in Haiti

File: configuration/concepts/misc.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
-,,,All Orderables,All Orderables,,ConvSet,N/A,,,Lab Samples; Radiology Orders,,,,,,,,
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Fully specified name:fr,Short name:fr,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1300
+,,,All Orderables,All Orderables,,All Orderables,All Orderables,,ConvSet,N/A,,,Echantillons de laboratoire; Ordonnances de radiologie,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/misc.csv]: Fix the uuid as done in Haiti

**Review by reagan-meant — CHANGES_REQUESTED** [review]
Lets fix the uuids

File: configuration/concepts/misc.csv (hunk 2 above)
Comment (nuhuhmutebi) [reply configuration/concepts/misc.csv]: Done

File: configuration/concepts/activate_forms.csv (hunk 1 above)
Comment (nuhuhmutebi) [reply configuration/concepts/activate_forms.csv]: Done

**Comment by reagan-meant** [comment]
Looks good.Thanks @nuhuhmutebi

**Pull request merged**